modules/aioseop_sitemap.php

```diff
@@ -862,7 +868,19 @@ function get_child_sitemap_urls() {
 					}
 				}
 			}
+
+			/**
+			 * Filter to add more URLs to the sitemap. Legacy!
+			 *
+			 * @param array	$siteurls	The string array of URLs.
```

Comment:
grunt will take care of this eventually.

Reply:
I still feel it would be wiser to use spaces when coded by the dev(s), and for a number of reasons too.

* Eventually doesn't resolve current situations.
    * ...and it seems like there's problems with making Grunt cohesive to the team members; which is also a concern.
* Tabs also don't align correctly, and some even misalign (as you can see in git); which makes it uneasy to read.
* In development, this will make inspecting code difficult when there's over 100 flags (which is the global limit for PHP Storm).
   * ...and it counts every tab used instead as a flag.
* I don't think it is wise to rely on a computer to do the work, but rather correct anything that got missed....and that is IF is catches it.

Reply:
@EkoJR I've pushed in changes to address this. Hesitation to adopt grunt is more a problem with priority than technology especially after having created a standalone instance. Hopefully, it will become a priority sooner than later as the code is becoming messier every iteration.

Reply:
@contactashish13 

I think some of us still need help in adopting grunt. :)